src/Microsoft.Management.UI.Internal/HelpWindow/ParagraphBuilder.cs

```diff
@@ -276,15 +276,8 @@ private static void MoveSpanToPosition(ref int currentSpanIndex, ref TextSpan? c
         /// <param name="length">Highlight length.</param>
         private void AddHighlight(int start, int length)
         {
-            if (start < 0)
-            {
-                throw new ArgumentOutOfRangeException("start");
-            }
-
-            if (start + length > this.textBuilder.Length)
-            {
-                throw new ArgumentOutOfRangeException("length");
-            }
+            ArgumentOutOfRangeException.ThrowIfNegative(start);
```

Comment:
I don't understand why the parameter name is no longer provided:
```suggestion
            ArgumentOutOfRangeException.ThrowIfNegative(start, nameof(start));
```

Reply:
It's no longer needed:
```cs
int start = -1;

if (start < 0)
{
	throw new ArgumentOutOfRangeException("start");
}
// Unhandled exception. System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'start')

ArgumentOutOfRangeException.ThrowIfNegative(start);
// Unhandled exception. System.ArgumentOutOfRangeException: 'start' must be a non-negative value. (Parameter 'start')
// Actual value was -1.
```